src/core/geometry/qgsgeometryutils.cpp

```diff
@@ -1366,14 +1366,19 @@ QPair<Qgis::WkbType, QString> QgsGeometryUtils::wktReadBlock( const QString &wkt
 {
   QString wktParsed = wkt;
   QString contents;
-  if ( wkt.contains( QLatin1String( "EMPTY" ), Qt::CaseInsensitive ) )
+  const QString empty { "EMPTY" };
```

Comment:
```suggestion
  const QString empty { QStringLiteral( "EMPTY" ) };
```

Reply:
Done, with `QLatin1String` since `QStringLiteral` is deprecated.